deployments/quick-start/kubernetes-vault-rbac.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-vault
+  namespace: default
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: kube-vault
+rules:
+- apiGroups: [""]
+  resources:
+  - pods
+  verbs: ["get", "list"]
```

Comment:
awesome! exactly what i needed, are these the full extend of apis, resources and verbs?